ir/attrs.h

```diff
@@ -9,6 +9,7 @@ namespace IR {
 
 class ParamAttrs final {
   unsigned bits;
+  uint64_t align;      // power-of-2 in bytes
```

Comment:
It was uint64_t, but char type would be sufficient for this; I'll address this in this PR if okay.